kiwi/exceptions.py

```diff
@@ -131,6 +131,10 @@ class KiwiDataStructureError(KiwiError):
     pass
 
 
+class KiwiDescriptionConflict(KiwiError):
+    pass
```

Comment:
Instead of `pass` I would just add a documentation here. Something like:
```python
# No
class KiwiDescriptionConflict(KiwiError):
    pass

# Yes
class KiwiDescriptionConflict(KiwiError):
    """
    Documentation, that describes a purpose of this class.
    Below of this documentation string there is nothing else.
    """
```